api/controllers/health_controller.go

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	"context"
+	"net/http"
+
+	responses "github.com/PTSS-Support/identity-service/api/dtos/responses/health"
+	"github.com/PTSS-Support/identity-service/core/services"
+	HealthStatus "github.com/PTSS-Support/identity-service/domain/enums"
+	"github.com/gin-gonic/gin"
+)
+
+type HealthController struct {
+	healthService *services.HealthService
+}
+
+type healthCheck func(ctx context.Context) (*responses.HealthResponse, error)
+
+// Due to this being a standardized, never-changing endpoint, we don't need a facade layer, this will only add unnecessary code to maintain.
+func NewHealthController(healthService *services.HealthService) *HealthController {
+	return &HealthController{
+		healthService: healthService,
+	}
+}
+
+func (c *HealthController) RegisterRoutes(router *gin.Engine) {
+	router.GET("/q/health", c.getHealth)
+	router.GET("/q/health/live", c.getLiveness)
+	router.GET("/q/health/ready", c.getReadiness)
+}
+
+func (c *HealthController) getHealth(ctx *gin.Context) {
+	c.handleHealthCheck(ctx, c.healthService.CheckReadiness)
```

Comment:
why does both health and readiness use the checkreadiness method?
Health endpoint should just say if pod is running so i'd say just make it so you respond with an object like:
```
{"status": "UP"}
```

Reply:
Idd, was een oversight die niet overeenkomt met smallrye-health, ik ga effe een CheckHealth method maken die de resultaten combineert in /q/health.